diff --git a/Makefile b/Makefile
index 8afa3f3..d573216 100644
--- a/Makefile
+++ b/Makefile
@@ -28,22 +28,22 @@ languages:
 install:
 	# install exec files
 	install -d $(DESTDIR)/usr/
-	cp -a sbin bin $(DESTDIR)/usr/
+	cp -dr sbin bin $(DESTDIR)/usr/
 	rm -f $(DESTDIR)/usr/sbin/Makefile
 	# install setup dir
 	install -d $(DESTDIR)/$(SHAREDIR)/
-	cp -a setup $(DESTDIR)/$(SHAREDIR)/
+	cp -dr setup $(DESTDIR)/$(SHAREDIR)/
 	# install other shared files
-	cp -a lang doc pkg pki image prerun postrun scripts/sbin scripts/bin $(DESTDIR)/$(SHAREDIR)/
+	cp -dr lang doc pkg pki image prerun postrun scripts/sbin scripts/bin $(DESTDIR)/$(SHAREDIR)/
 	# erase an extra COPYING
 	rm -f $(DESTDIR)/$(SHAREDIR)/pkg/misc/COPYING
 	rm -f $(DESTDIR)/$(SHAREDIR)/pkg/misc/Makefile $(DESTDIR)/$(SHAREDIR)/lang/Makefile
 	# install config files
 	install -d $(DESTDIR)/etc/drbl/
-	cp -a conf/* $(DESTDIR)/etc/drbl/
+	cp -dr conf/* $(DESTDIR)/etc/drbl/
 	# install themes
 	install -d $(DESTDIR)/usr/share/gdm/themes/drbl-gdm
-	cp -a themes/* $(DESTDIR)/usr/share/gdm/themes/drbl-gdm/
+	cp -dr themes/* $(DESTDIR)/usr/share/gdm/themes/drbl-gdm/
 
 clean:
 	$(MAKE) -C sbin clean
